test/data/testSP.py

```diff
@@ -241,19 +241,10 @@ def testmoments(self):
                 self.assertEquals(m, 0.0)
 
     @skipForParser('ADF', 'Does not support metadata yet')
-    @skipForParser('GAMESS', 'Does not support metadata yet')
     @skipForParser('GAMESSUK', 'Does not support metadata yet')
-    @skipForParser('Gaussian', 'Does not support metadata yet')
-    @skipForParser('Jaguar', 'Does not support metadata yet')
-    @skipForParser('Molpro', 'Does not support metadata yet')
-    @skipForParser('NWChem', 'Does not support metadata yet')
-    @skipForParser('ORCA', 'Does not support metadata yet')
-    @skipForParser('Psi', 'Does not support metadata yet')
-    @skipForParser('QChem', 'Does not support metadata yet')
     def testmetadata(self):
         """Does metadata have expected keys and values?"""
         self.assertTrue(hasattr(self.data, "metadata"))
-        self.assertIn("basis_set", self.data.metadata)
         self.assertIn("methods", self.data.metadata)
         self.assertIn("package", self.data.metadata)
         self.assertIn("package_version", self.data.metadata)

```

Comment:
It is related to the metadata parsing as a basis set may not be named in the metadata. Some programs will set a default basis set while for others there are many various different ways to read in a basis set. This can be difficult to properly parse (e.g. in PSI4 you can do `set basis cc-pVTZ` or `energy('CCSD(T)/cc-pVTZ')`) and can mean that there is non basis set name (e.g. entering parameters directly into input file). Furthermore, different basis sets can be specified for different atoms, thus there would be multiple basis sets in use.